src/udev/udevd.c

```diff
@@ -65,70 +66,37 @@ static int listen_fds(int *ret_ctrl, int *ret_netlink) {
         return 0;
 }
 
+static DEFINE_CONFIG_PARSE_ENUM(config_parse_resolve_name_timing, resolve_name_timing, ResolveNameTiming, "Failed to parse resolve name timing");
+
 static int manager_parse_udev_config(Manager *manager) {
-        _cleanup_free_ char *log_val = NULL, *children_max = NULL, *exec_delay = NULL,
-                *event_timeout = NULL, *resolve_names = NULL, *timeout_signal = NULL;
+        int log_val = -1;
+        Manager *m = ASSERT_PTR(manager); /* A hack to check the ptr before using it in the table */
```

Comment:
Why not use normal assertion?

Reply:
> Why not use normal assertion?

We don't mix `assert`s and declarations.

I could have used `ASSERT_PTR()` just in the first line of the dispatch table (or even in all lines), but the current variant with an intermediate variable appeared slightly nicer to me.

Reply:
> We don't mix asserts and declarations.

You can define the table after variable declarations and assertions, as with `shared/sleep-config.c`.

Reply:
Fixed.